src/webui/www/private/newcategory.html

```diff
@@ -74,6 +135,7 @@
                         fetch("api/v2/torrents/createCategory", {
                                 method: "POST",
                                 body: new URLSearchParams({
+                                    ...body,
```

Comment:
I would suggest renaming the variable to avoid this little confusing pattern:
```suggestion
                                body: new URLSearchParams({
                                    ...dlPathParams,
```